AUTHORS.txt

```diff
@@ -2,7 +2,6 @@ Lukáš Lalinský <lalinsky@gmail.com>
 Robert Kaye <rob@eorbit.net>
 Philipp Wolfer <ph.wolfer@googlemail.com>
 
-Picard icons by Carlin Mangar <openshift@gmail.com>
-
-The icons in this program are from the Tango Desktop Project
-<http://tango.freedesktop.org/>.
+Picard logo by <http://monkeydo.digital>, © 2015 MetaBrainz Foundation
+Some icons in this program are by Carlin Mangar <openshift@gmail.com>, the
```

Comment:
From memory: He created the logo and the empty CD case shown if no cover art is availableand the AcoustId (back then it were still PUIDs) submission icon as it is now (this was an updated version of the existing icon, see https://github.com/musicbrainz/picard/commit/3faab92ef96707ff1e68c50b61df403864c49325)

@Freso Was there anything else you saw?